tools/const_finder/lib/const_finder.dart

```diff
@@ -31,10 +33,25 @@ class _ConstVisitor extends RecursiveVisitor<void> {
   final List<Map<String, dynamic>> nonConstantLocations;
 
   bool _matches(Class node) {
-    return node.enclosingLibrary.canonicalName.name == classLibraryUri &&
+    return node.enclosingLibrary.importUri.toString() == classLibraryUri &&
```

Comment:
This is more correct than using canonicalName, per @alexmarkov offline discussion.